.clang-tidy

```diff
@@ -3,6 +3,7 @@ Checks: >
   *,
   -abseil-*,
   -altera-*,
+  -boost-*,
```

Comment:
Note that I disabled this because there's a range of new checks that suggest replacing `std::*` algorithms with their `boost::*` equivalent, which would just bloat our compile times.